client/src/components/layout/AccountPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import _ from "lodash";
+import Dropzone from "react-dropzone";
+
+const AccountPage = (props) => {
+ 
+  const [user, setUser] = useState(props.user); 
+  const [profilePicture, setProfilePicture] = useState({}); 
+  const handleImageUpload = (acceptedImage) => {
+    setProfilePicture({
+      ...profilePicture,
+      image: acceptedImage[0],
+    });
+  };
+
+  const addProfilePicture = async () => {
+    const newProfilePictureBody = new FormData();
+    newProfilePictureBody.append("image", profilePicture.image);
+    try {
+      const response = await fetch(`/api/v1/users/${profilePicture.id}`, {
+        method: "PATCH",
+        headers: {
+          Accept: "image/jpeg",
+        },
+        body: newProfilePictureBody,
+      });
+      if (!response.ok) {
+        throw new Error(`${response.status} (${response.statusText})`);
+      }
+      const body = await response.json();
+      setUser({ ...user, image: body.user.image });
+    } catch (error) {
+      console.error(`Error in addProfilePicture Fetch: ${error.message}`);
+    }
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    addProfilePicture();
+  };
+
+  let uploadImagePromptText = "Click Or Drag Here To Edit Profile Image";
+
+  if (!_.isEmpty(profilePicture)) {
+    uploadImagePromptText = "Image received. Click button below to save!";
```

Comment:
if you're interested, here is an example of how you can display an image preview once an image has been uploaded: https://github.com/LaunchAcademy/node-react-file-upload/blob/main/client/src/components/memes/NewMemeForm.js